plenum/common/txn.py

```diff
@@ -80,6 +81,10 @@
 LAST_TXN = "lastTxn"
 TXNS = "Txns"
 
+STEWARD = Roles.STEWARD.value
+TRUST_ANCHOR = Roles.TRUST_ANCHOR.value
+TRUSTEE = Roles.TRUSTEE.value
+TGB = Roles.TGB.value
```

Comment:
Isn't this going to result in an inconsistency in our ledger, where some transactions are identified by a symbolic name like "NYM", and some are represented by numbers (e.g., Roles.TRUSTEE.value)? Would we want to create a numeric constant for all transactions, not just the ones that create a role?